index.js

```diff
@@ -151,6 +154,21 @@ async function register (server, options) {
     logger.info(server.info, 'server stopped')
   })
 
+  function isLoggingIgnored (options, request) {
```

Comment:
Please pass in `ignorePaths` directly.

Reply:
`options` is being used twice here. `options.ignoreTags` and `options.ignorePaths`, do you still prefer to pass both options instead?

Reply:
It's generally better to not access options at runtime because the object can change from the outside.

Reply:
It's ok, no problem